usr/src/test/smartos-test/README

```diff
@@ -130,9 +136,11 @@ inconsistencies in the BHYVE/VMM results) which has:
 	A 100GB disk for `zones` (a smaller disk can cause ENOSPC problems
 	sometimes)
 
-	Three 30GB disks for setting up in DISKS for ZFS testing.  (fewer
+	Six 30GB disks for setting up in DISKS for ZFS testing.  (fewer
 	can cause ZFS test failures)
 
+	(optional) a Virtual NVMe drive of 20GB.
```

Comment:
> of 20GB.

...or greater size?

Reply:
I'm honestly not sure if 20GB is too BIG, but out of caution, I will use your suggestion.